src/Miunie.Storage/JsonDataStorage.cs

```diff
@@ -76,39 +70,24 @@ public void WipeData()
             Directory.Delete(_resourcesFolder);
         }
 
-        public bool KeyExists(string collection, string key)
-        {
-            var file = GetFileNameByKey(key);
-            return LocalFileExists(String.Concat(collection, "/", file));
-        }
+        private string KeyToFullFileName(string key) => String.Format(FileTemplate, key);
 
-        private bool LocalFileExists(string file)
-        {
-            string filePath = String.Concat(_resourcesFolder, "/", file);
-            return File.Exists(filePath);
-        }
+        private string GetFullFilePath(string collection, string key) =>

```

Comment:
Same as before, let's keep it consistent (: `=>` in a new line.

Reply:
_For a "first commit", it's common to apply these simple fixes and improvements yourself, @Charly6596. So keep that in mind._